view/theme/frio/templates/settings_features.tpl

```diff
@@ -19,7 +19,11 @@
 				<div id="{{$g}}-settings-content" class="panel-collapse collapse" role="tabpanel" aria-labelledby="{{$g}}-settings-title">
 					<div class="section-content-tools-wrapper">
 						{{foreach $f.1 as $fcat}}
+							{{* Here is an addional condition to filter the richtext feature
+							because frio doesn't support the richtext *}}
+							{{if $fcat.0 != 'feature_richtext'}}
```

Comment:
My idea is to fix the script.
mainly, I want to try to let the script use smarty3 someway to compile the templates in a temporary directory tree and then use standard xgettext to extract language strings..

(edit: comment in the right thread...)